conf/interpreter-list

```diff
@@ -35,3 +35,4 @@ md              org.apache.zeppelin:zeppelin-markdown:0.6.1             Markdown
 postgresql      org.apache.zeppelin:zeppelin-postgresql:0.6.1           Postgresql interpreter
 python          org.apache.zeppelin:zeppelin-python:0.6.1               Python interpreter
 shell           org.apache.zeppelin:zeppelin-shell:0.6.1                Shell command
+scio            org.apache.zeppelin:zeppelin-scio:0.6.1                 Scio interpreter
```

Comment:
@ravwojdyla can we keep maintaining alphabetic order in here? need to change the position of `scio` with `shell`.